### ISSUE TITLE
Deprecation warnings on 1.6

### ISSUE THREAD

**Issue by supersimple** [igas/faker#138]
These are all the warnings I get when compiling on Erlang 20.3 Elixir 1.6.3
```
warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/color.ex:22

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/color.ex:23

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/color.ex:24

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/date.ex:26

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/date.ex:27

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/date.ex:29

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/commerce.ex:27

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/beer.ex:51

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/beer.ex:71

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/beer.ex:71

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/file.ex:100

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/datetime.ex:68

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/code/iban.ex:120

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker.ex:35

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker.ex:52

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/util.ex:11

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/phone/en_us.ex:84

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet/en.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet/en.ex:18

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/team/en.ex:19

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/color/en.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/color/en.ex:18

warning: String.rjust/3 is deprecated, use String.pad_leading/3 with a binary padding
  lib/faker/internet.ex:119

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pokemon/en.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pokemon/en.ex:18

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem/shakespeare/ru.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem/shakespeare/ru.ex:18

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem/shakespeare/ru.ex:24

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem/shakespeare/ru.ex:30

warning: String.rjust/3 is deprecated, use String.pad_leading/3 with a binary padding
  lib/faker/internet.ex:161

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem/shakespeare/en.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem/shakespeare/en.ex:18

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem/shakespeare/en.ex:24

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem/shakespeare/en.ex:30

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/app.ex:17

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/app.ex:37

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/app.ex:45

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/app.ex:53

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/app.ex:60

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/beer/en.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/beer/en.ex:18

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/beer/en.ex:24

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/beer/en.ex:30

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/beer/en.ex:36

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:30

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:33

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:81

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:92

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:95

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:95

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:96

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:96

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:97

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:97

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:105

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:115

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet.ex:157

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/nato.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/nato.ex:18

warning: key "Geographic numbers with area codes - for list see telephone area codes" will be overridden in map
  lib/faker/phone/en_gb.ex:9

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/commerce/en.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/commerce/en.ex:18

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/commerce/en.ex:26

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/commerce/en.ex:39

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/commerce/en.ex:45

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/commerce/en.ex:51

warning: String.to_char_list/1 is deprecated, use String.to_charlist/1
  lib/faker/phone/en_gb.ex:67

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/superhero/en.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/superhero/en.ex:24

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/superhero/en.ex:32

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/superhero/en.ex:41

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/superhero/en.ex:102

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet/user_agent.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet/user_agent.ex:20

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet/user_agent.ex:30

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet/user_agent.ex:37

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet/user_agent.ex:47

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet/user_agent.ex:57

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/internet/user_agent.ex:68

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/phone/en_gb.ex:34

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/phone/en_gb.ex:50

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/phone/en_gb.ex:77

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/phone/en_gb.ex:78

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/company/en.ex:20

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/company/en.ex:26

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/company/en.ex:32

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/company/en.ex:38

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/company/en.ex:44

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/company/en.ex:50

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/company/en.ex:62

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/company/en.ex:73

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem.ex:22

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem.ex:46

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem.ex:67

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem.ex:92

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem.ex:126

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem.ex:150

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/lorem.ex:168

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:21

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:34

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:51

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:68

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:78

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:84

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:90

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:96

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:102

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:108

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:114

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:120

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:128

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/pizza.ex:134

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/name/en.ex:12

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/name/en.ex:24

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/name/en.ex:30

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/name/en.ex:36

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/name/en.ex:42

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/name/en.ex:56

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/name/en.ex:62

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/name/en.ex:68

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:16

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:25

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:37

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:43

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:49

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:55

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:87

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:95

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:101

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:123

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:133

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:139

warning: crypto:rand_uniform/2 is deprecated and will be removed in a future release; use rand:uniform/1
  lib/faker/address.ex:153
```

**Comment by jakobdamjensen** [comment]
I was going to fix this and submit a PR but It seems to have been fixed in master already. It just needs to be pushed to hex AFAIK.

**Comment by igas** [comment]
Hi everyone, I published v0.10.0. Please test.
